impeller/entity/contents/filters/inputs/filter_input.h

```diff
@@ -77,6 +77,12 @@ class FilterInput {
   ///         of `inputs`.
   /// @see    `FilterInput::IsLeaf`
   virtual void SetLeafInputs(const FilterInput::Vector& inputs);
+
+  /// @brief  Sets the effect transform of filter inputs.
+  virtual void SetEffectTransform(const Matrix& matrix);
+
+  /// @brief  Turns on subpass mode for filter inputs.
+  virtual void SetIsForSubpass(bool is_for_subpass);
```

Comment:
Nit: can you add more context on what subpass mode implies?

Reply:
Done.